pages/dkp/kaptain/1.3.0/uninstall/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: layout.pug
+navigationTitle: Uninstall
+title: Uninstall Kaptain
+menuWeight: 7
+excerpt: Uninstall Kaptain from your cluster
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>When uninstalling Kaptain, all data, the current state, and configuration of your application will be lost.</p>
+
+## Uninstall Kaptain
+
+Uninstalling Kaptain requires the execution of some manual steps in addition to the regular "KUDO uninstall" command.
+
+1.  Delete all profiles configured with Kaptain. Profiles have dependencies (such as finalizers) associated with Kaptain, and removing Kaptain before removing your profiles will put your cluster in a broken state.
+
+    The following command deletes all notebooks, pipelines and deployed models. Notebook volumes created when deploying notebooks are cleaned up and deleted. **Ensure you have backed up** anything that is not in managed storage (even if model artifacts are stored in your object store, and your notebook code is in version control).
+
+    ```bash
+    kubectl get profile --output name | while read name; do
+      kubectl delete $name
+    done
+    ```
+
+**DO NOT** proceed until all profiles are removed. If a profile cannot be removed, review the troubleshooting section for instructions to [forcibly delete a profile][#cannot-finalize-removing-a-profile].
+
+1.  Uninstall Kaptain using KUDO. This command deletes the Kaptain Percona DB store, all deployments, statefulsets, jobs, and volumes associated with the kubeflow namespace.
+
+    ```
+    kubectl kudo uninstall --instance "kaptain" --namespace kubeflow --wait --wait-time=600
+    ```
+
+1.  Ensure that the following namespaces are removed:
+
+    ```bash
+    for ns in kubeflow kaptain-ingres kserve; do
+      kubectl delete namespace $ns
+    done
+    ```
+
+1.  Clean up several additional configuration resources left behind by KUDO-managed Kaptain, so you can install Helm-managed Kaptain correctly. This command selects various resource types with the label `kudo.dev/instance` of either `"kaptain"` or beginning with `"kaptain-"`.
+
+    ```bash
+    for type in clusterrole customresourcedefinition clusterrolebinding clusterrole mutatingwebhookconfiguration ValidatingWebhookConfiguration; do
+      INSTANCES=$(kubectl get $type  --output json | jq '.items[].metadata.labels["kudo.dev/instance"] | select(. != null)'  -r | egrep '^kaptain(-|$)' | tr "\n" ",")
+      kubectl delete $type --selector "kudo.dev/instance in ($INSTANCES)"
+    done
+    ```
+
+    Kaptain is now fully uninstalled.
+
+1.  Remove Kudo:
+
+    ```bash
+    kubectl kudo init --upgrade --dry-run --output yaml | kubectl delete -f -
+    ```
+
+## Troubleshooting and overcoming failures
+
+Sometimes, uninstall steps fail or hang because a cleanup task gets stuck. In that case, force Kubernetes to uninstall resources. Some cleanup tasks may not be fully run.
```

Comment:
```suggestion
Sometimes, the uninstall steps will fail or hang because a cleanup task gets stuck. In that case, you will need to force Kubernetes to uninstall the resources.  In such cases, you may need to clean up things manually.  Follow the steps below as necessary.
```